lib/ao_host.js

```diff
@@ -661,30 +663,24 @@ class AOHost extends AsyncEventEmitter {
       return
     }
 
+    const serialized = this.getSerializedAO(instance)
+    await this.emit('ao:state:update', serialized)
+  }
+
+  getSerializedAO (instance) {
     const { state = {} } = instance
-    const { id } = state
+    const { id, gid } = state
     const ao = this.getAO(id)
 
-    if (!ao) {
-      throw new Error(`can\t persist unknown ao: ${id}`)
-    }
-
     const { meta = {} } = ao
     const { serialize } = meta
 
-    if (!serialize) {
-      debug('can\t save AO %s [%s] due to missing serialize method', gid, id)
-      return
-    }
-
-    await this.emit('ao:persist:db:update', {
```

Comment:
`ao:persist:db:update` also needs to be removed from README.md

Reply:
fixed!